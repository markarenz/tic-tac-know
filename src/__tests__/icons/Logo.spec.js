import { render, screen } from '@testing-library/react';
import Logo from '../../icons/Logo';

describe('Logo', () => {
  it('renders component', () => {
    const container = render(<Logo />);
    expect(container).toMatchSnapshot();
  });
});
