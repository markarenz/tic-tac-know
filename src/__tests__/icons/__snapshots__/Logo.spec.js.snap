// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        viewBox="0 0 100 50"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="#fff"
          stroke="#000"
          stroke-opacity="0"
          transform="translate(0 -4.763)"
        >
          <g
            font-family="Roboto Slab"
            font-size="47.978"
            font-weight="bold"
            stroke-width="0.529"
            transform="matrix(.61769 0 0 .72505 -.785 -12.51)"
          >
            <path
              d="M13.025 82.729l-3.538-4.053 16.704-17.219.046-.07-3.373-.281v-4.123h14.033v4.123l-3.21.632zm-9.043 8.363v-4.1l3.631-.702V61.809l-3.63-.703v-4.123H17.92v4.123l-3.467.703v24.48l3.467.703v4.1zm19.632 0v-4.1l3.186-.28h.023l-8.668-11.948 4.522-4.522 12.088 16.165 3.35.585v4.1z"
              style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
            />
            <path
              d="M40.176 91.092v-4.1l3.631-.702V61.809l-3.63-.703v-4.123h10.565l14.852 23.66.14-.023V61.81l-4.357-.703v-4.123H75.504v4.123l-3.655.703v29.283h-6.606l-15.18-23.427-.141.024v18.6l4.38.703v4.1z"
              style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
            />
            <path
              d="M122.521 91.092l-7.918-29.635-3.49-.351v-4.123h13.727v4.123l-3.678.539 4.006 16.586.726 3.045h.141l.867-3.045 6.747-21.248h5.036l7.005 21.833.82 2.695h.14l.703-2.695 4.147-17.171-3.89-.54v-4.122h13.471v4.123l-3.256.351-8.012 29.635h-6.185l-6.465-19.936-.82-2.811h-.14l-.914 2.905-6.537 19.842z"
              style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
            />
          </g>
          <g
            transform="matrix(.90098 0 0 .75093 -2.28 4.891)"
          >
            <path
              d="M10.318 27.592v-1.499l2.808-.223V4.2H6.081l-.224 3.824H4.169V2.512h19.964v5.512h-1.67L22.22 4.2h-7.028v21.67l2.808.223v1.5z"
              fill="#c084fc"
              style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
            />
            <path
              d="M25.23 27.592v-1.499l2.807-.223V10.694l-2.807-.224V8.954h4.874V25.87l2.808.223v1.5zM27.796 3.7V.72h2.308V3.7z"
              fill="#c084fc"
              style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
            />
            <path
              d="M42.666 27.954q-2.326 0-4.117-1.172-1.775-1.17-2.79-3.272-1-2.102-1-4.875v-.724q0-2.721.965-4.823.982-2.101 2.756-3.29 1.774-1.188 4.168-1.188 2.17 0 3.859.826 1.688.81 2.825 2.102l.034 4.22h-1.722l-.276-3.48q-.861-.912-2.015-1.412-1.154-.517-2.653-.517-1.757 0-3.083.982-1.327.982-2.067 2.705-.724 1.705-.724 3.875v.724q0 2.274.724 3.996.723 1.705 2.032 2.653 1.31.947 3.1.947 2.085 0 3.48-1.085 1.396-1.102 1.688-3.238h1.844l.034.103q-.155 1.757-1.05 3.1-.88 1.344-2.412 2.102-1.516.74-3.6.74z"
              fill="#c084fc"
              style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
            />
            <path
              d="M52.815 17.756v-1.722h8.251v1.722z"
              fill="#c084fc"
              style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
            />
            <path
              d="M65.766 27.592v-1.499l2.808-.223V4.2h-7.046l-.224 3.824h-1.688V2.512h19.965v5.512H77.91L77.669 4.2H70.64v21.67l2.807.223v1.5z"
              fill="#c084fc"
              style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
            />
            <path
              d="M84.907 27.954q-2.653 0-4.151-1.413-1.499-1.412-1.499-3.824 0-1.722 1-3.014.998-1.31 2.824-2.033 1.843-.723 4.358-.723h3.859V14.81q0-2.016-1.206-3.239-1.189-1.24-3.6-1.24-1.361 0-2.446.448-1.068.43-1.86 1.102l-.173 2.843h-1.705v-3.893q1.206-1.034 2.739-1.62 1.533-.602 3.48-.602 3.29 0 5.064 1.653 1.774 1.654 1.774 4.582v8.992q0 .551.034 1.102.035.534.12 1.068l2.223.086v1.5h-4.065q-.207-1.017-.293-1.637-.07-.62-.086-1.396-1.017 1.499-2.687 2.446-1.671.948-3.704.948zm.241-1.809q1.912 0 3.618-.999 1.722-1.016 2.532-2.687V18.53h-3.876q-2.86 0-4.479 1.24-1.619 1.224-1.619 3.015 0 1.499 1.034 2.429 1.033.93 2.79.93z"
              fill="#c084fc"
              style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
            />
            <path
              d="M104.45 27.954q-2.326 0-4.118-1.172-1.774-1.17-2.79-3.272-1-2.102-1-4.875v-.724q0-2.721.965-4.823.982-2.101 2.756-3.29 1.775-1.188 4.17-1.188 2.17 0 3.858.826 1.688.81 2.825 2.102l.034 4.22h-1.723l-.275-3.48q-.861-.912-2.016-1.412-1.154-.517-2.652-.517-1.757 0-3.084.982-1.326.982-2.067 2.705-.723 1.705-.723 3.875v.724q0 2.274.723 3.996.724 1.705 2.033 2.653 1.309.947 3.1.947 2.085 0 3.48-1.085 1.395-1.102 1.688-3.238h1.843l.035.103q-.155 1.757-1.051 3.1-.878 1.344-2.412 2.102-1.515.74-3.6.74z"
              fill="#c084fc"
              style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
            />
          </g>
          <g
            transform="matrix(.7033 0 0 .84119 4.282 -13.468)"
          >
            <path
              d="M60.318 59.318H89.681V60.681H60.318z"
              stroke-width="0.637"
            />
            <path
              d="M60.318 69.318H89.681V70.681H60.318z"
              stroke-width="0.637"
            />
            <path
              d="M-79.682 69.318H-50.319V70.681H-79.682z"
              stroke-width="0.637"
              transform="rotate(-90)"
            />
            <path
              d="M-79.682 79.318H-50.319V80.681H-79.682z"
              stroke-width="0.637"
              transform="rotate(-90)"
            />
            <path
              class="animate-logoTicA opacity-0"
              d="M73.147 62.156l1.696 2.112 1.687-2.112h1.446l-2.371 2.901 2.294 2.821h-1.446l-1.61-2.03-1.62 2.03h-1.455l2.294-2.82-2.37-2.902z"
              font-family="Raleway"
              font-size="47.978"
              font-weight="bold"
              id="x5"
              stroke-width="0.097"
            />
            <path
              class="animate-logoTicB opacity-0"
              d="M64.8 77.913q-.75 0-1.368-.242-.607-.242-1.05-.653-.434-.419-.675-.935-.241-.523-.241-1.072 0-.572.25-1.088.26-.523.704-.926.453-.411 1.06-.645.617-.242 1.35-.242.742 0 1.35.25.616.25 1.05.669.434.419.675.935.24.516.24 1.064 0 .564-.25 1.088-.25.515-.704.926-.443.403-1.06.637-.607.234-1.33.234zm-1.985-2.902q0 .371.135.718.135.346.386.62.26.266.626.427.376.154.849.154.482 0 .857-.162.376-.169.627-.443.25-.282.376-.62.135-.347.135-.694 0-.37-.145-.709-.135-.346-.395-.612-.25-.274-.626-.427-.367-.162-.83-.162-.491 0-.867.17-.366.16-.617.435-.25.274-.385.612-.126.339-.126.693z"
              font-family="Raleway"
              font-size="47.978"
              font-weight="bold"
              id="o7"
              stroke-width="0.097"
            />
            <path
              class="animate-logoTicC opacity-0"
              d="M63.119 51.863l1.696 2.111 1.687-2.111h1.446l-2.371 2.901 2.294 2.82h-1.446l-1.61-2.03-1.619 2.03H61.74l2.295-2.82-2.372-2.901z"
              font-family="Raleway"
              font-size="47.978"
              font-weight="bold"
              id="x1"
              stroke-width="0.097"
            />
            <path
              class="animate-logoTicD opacity-0"
              d="M85.183 77.913q-.752 0-1.369-.242-.607-.242-1.05-.653-.434-.419-.675-.935-.24-.523-.24-1.072 0-.572.25-1.088.26-.523.703-.926.453-.411 1.06-.645.618-.242 1.35-.242.742 0 1.35.25.617.25 1.05.669.434.419.675.935.241.516.241 1.064 0 .564-.25 1.088-.251.515-.704.926-.444.403-1.06.637-.608.234-1.33.234zm-1.985-2.902q0 .371.135.718.134.346.385.62.26.266.627.427.376.154.848.154.482 0 .858-.162.376-.169.626-.443.251-.282.376-.62.135-.347.135-.694 0-.37-.144-.709-.135-.346-.396-.612-.25-.274-.626-.427-.366-.162-.83-.162-.49 0-.867.17-.366.16-.617.435-.25.274-.385.612-.125.339-.125.693z"
              font-family="Raleway"
              font-size="47.978"
              font-weight="bold"
              id="o9"
              stroke-width="0.097"
            />
            <path
              class="animate-logoTicE opacity-0"
              d="M73.147 72.11l1.696 2.112 1.687-2.112h1.446l-2.371 2.902 2.294 2.82h-1.446l-1.61-2.03-1.62 2.03h-1.455l2.294-2.82-2.37-2.902z"
              font-family="Raleway"
              font-size="47.978"
              font-weight="bold"
              id="x8"
              stroke-width="0.097"
            />
            <path
              class="animate-logoTicF opacity-0"
              d="M74.829 57.665q-.752 0-1.369-.241-.607-.242-1.05-.653-.434-.42-.675-.935-.241-.524-.241-1.072 0-.572.25-1.088.26-.524.704-.927.453-.41 1.06-.645.617-.241 1.35-.241.742 0 1.35.25.616.25 1.05.668.434.42.675.935.24.516.24 1.064 0 .564-.25 1.088-.25.516-.704.927-.443.403-1.06.637-.607.233-1.33.233zm-1.986-2.901q0 .37.135.717t.386.62q.26.267.626.428.376.153.848.153.482 0 .858-.161.376-.17.627-.443.25-.282.376-.62.135-.347.135-.694 0-.37-.145-.71-.135-.346-.395-.612-.25-.274-.627-.427-.366-.161-.829-.161-.491 0-.867.17-.366.16-.617.434-.25.274-.386.613-.125.338-.125.693z"
              font-family="Raleway"
              font-size="47.978"
              font-weight="bold"
              id="o2"
              stroke-width="0.097"
            />
            <path
              class="animate-logoTicG opacity-0"
              d="M83.501 62.156l1.697 2.112 1.687-2.112h1.445l-2.37 2.901 2.293 2.821h-1.446l-1.61-2.03-1.619 2.03h-1.455l2.294-2.82-2.371-2.902z"
              font-family="Raleway"
              font-size="47.978"
              font-weight="bold"
              id="x6"
              stroke-width="0.097"
            />
            <path
              class="animate-logoTicH opacity-0"
              d="M64.8 67.959q-.75 0-1.368-.242-.607-.242-1.05-.653-.434-.419-.675-.935-.241-.524-.241-1.072 0-.572.25-1.088.26-.523.704-.926.453-.411 1.06-.645.617-.242 1.35-.242.742 0 1.35.25.616.25 1.05.669.434.419.675.935.24.516.24 1.064 0 .564-.25 1.088-.25.515-.704.926-.443.403-1.06.637-.607.234-1.33.234zm-1.985-2.902q0 .371.135.718.135.346.386.62.26.266.626.427.376.154.849.154.482 0 .857-.162.376-.169.627-.443.25-.282.376-.62.135-.347.135-.694 0-.37-.145-.709-.135-.346-.395-.612-.25-.274-.626-.427-.367-.162-.83-.162-.491 0-.867.17-.366.16-.617.435-.25.274-.385.612-.126.339-.126.693z"
              font-family="Raleway"
              font-size="47.978"
              font-weight="bold"
              id="o4"
              stroke-width="0.097"
            />
            <path
              class="animate-logoTicI opacity-0"
              d="M83.501 51.863l1.697 2.111 1.687-2.111h1.445l-2.37 2.901 2.293 2.82h-1.446l-1.61-2.03-1.619 2.03h-1.455l2.294-2.82-2.371-2.901z"
              font-family="Raleway"
              font-size="47.978"
              font-weight="bold"
              id="x3"
              stroke-width="0.097"
            />
          </g>
        </g>
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      viewBox="0 0 100 50"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        fill="#fff"
        stroke="#000"
        stroke-opacity="0"
        transform="translate(0 -4.763)"
      >
        <g
          font-family="Roboto Slab"
          font-size="47.978"
          font-weight="bold"
          stroke-width="0.529"
          transform="matrix(.61769 0 0 .72505 -.785 -12.51)"
        >
          <path
            d="M13.025 82.729l-3.538-4.053 16.704-17.219.046-.07-3.373-.281v-4.123h14.033v4.123l-3.21.632zm-9.043 8.363v-4.1l3.631-.702V61.809l-3.63-.703v-4.123H17.92v4.123l-3.467.703v24.48l3.467.703v4.1zm19.632 0v-4.1l3.186-.28h.023l-8.668-11.948 4.522-4.522 12.088 16.165 3.35.585v4.1z"
            style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
          />
          <path
            d="M40.176 91.092v-4.1l3.631-.702V61.809l-3.63-.703v-4.123h10.565l14.852 23.66.14-.023V61.81l-4.357-.703v-4.123H75.504v4.123l-3.655.703v29.283h-6.606l-15.18-23.427-.141.024v18.6l4.38.703v4.1z"
            style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
          />
          <path
            d="M122.521 91.092l-7.918-29.635-3.49-.351v-4.123h13.727v4.123l-3.678.539 4.006 16.586.726 3.045h.141l.867-3.045 6.747-21.248h5.036l7.005 21.833.82 2.695h.14l.703-2.695 4.147-17.171-3.89-.54v-4.122h13.471v4.123l-3.256.351-8.012 29.635h-6.185l-6.465-19.936-.82-2.811h-.14l-.914 2.905-6.537 19.842z"
            style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
          />
        </g>
        <g
          transform="matrix(.90098 0 0 .75093 -2.28 4.891)"
        >
          <path
            d="M10.318 27.592v-1.499l2.808-.223V4.2H6.081l-.224 3.824H4.169V2.512h19.964v5.512h-1.67L22.22 4.2h-7.028v21.67l2.808.223v1.5z"
            fill="#c084fc"
            style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
          />
          <path
            d="M25.23 27.592v-1.499l2.807-.223V10.694l-2.807-.224V8.954h4.874V25.87l2.808.223v1.5zM27.796 3.7V.72h2.308V3.7z"
            fill="#c084fc"
            style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
          />
          <path
            d="M42.666 27.954q-2.326 0-4.117-1.172-1.775-1.17-2.79-3.272-1-2.102-1-4.875v-.724q0-2.721.965-4.823.982-2.101 2.756-3.29 1.774-1.188 4.168-1.188 2.17 0 3.859.826 1.688.81 2.825 2.102l.034 4.22h-1.722l-.276-3.48q-.861-.912-2.015-1.412-1.154-.517-2.653-.517-1.757 0-3.083.982-1.327.982-2.067 2.705-.724 1.705-.724 3.875v.724q0 2.274.724 3.996.723 1.705 2.032 2.653 1.31.947 3.1.947 2.085 0 3.48-1.085 1.396-1.102 1.688-3.238h1.844l.034.103q-.155 1.757-1.05 3.1-.88 1.344-2.412 2.102-1.516.74-3.6.74z"
            fill="#c084fc"
            style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
          />
          <path
            d="M52.815 17.756v-1.722h8.251v1.722z"
            fill="#c084fc"
            style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
          />
          <path
            d="M65.766 27.592v-1.499l2.808-.223V4.2h-7.046l-.224 3.824h-1.688V2.512h19.965v5.512H77.91L77.669 4.2H70.64v21.67l2.807.223v1.5z"
            fill="#c084fc"
            style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
          />
          <path
            d="M84.907 27.954q-2.653 0-4.151-1.413-1.499-1.412-1.499-3.824 0-1.722 1-3.014.998-1.31 2.824-2.033 1.843-.723 4.358-.723h3.859V14.81q0-2.016-1.206-3.239-1.189-1.24-3.6-1.24-1.361 0-2.446.448-1.068.43-1.86 1.102l-.173 2.843h-1.705v-3.893q1.206-1.034 2.739-1.62 1.533-.602 3.48-.602 3.29 0 5.064 1.653 1.774 1.654 1.774 4.582v8.992q0 .551.034 1.102.035.534.12 1.068l2.223.086v1.5h-4.065q-.207-1.017-.293-1.637-.07-.62-.086-1.396-1.017 1.499-2.687 2.446-1.671.948-3.704.948zm.241-1.809q1.912 0 3.618-.999 1.722-1.016 2.532-2.687V18.53h-3.876q-2.86 0-4.479 1.24-1.619 1.224-1.619 3.015 0 1.499 1.034 2.429 1.033.93 2.79.93z"
            fill="#c084fc"
            style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
          />
          <path
            d="M104.45 27.954q-2.326 0-4.118-1.172-1.774-1.17-2.79-3.272-1-2.102-1-4.875v-.724q0-2.721.965-4.823.982-2.101 2.756-3.29 1.775-1.188 4.17-1.188 2.17 0 3.858.826 1.688.81 2.825 2.102l.034 4.22h-1.723l-.275-3.48q-.861-.912-2.016-1.412-1.154-.517-2.652-.517-1.757 0-3.084.982-1.326.982-2.067 2.705-.723 1.705-.723 3.875v.724q0 2.274.723 3.996.724 1.705 2.033 2.653 1.309.947 3.1.947 2.085 0 3.48-1.085 1.395-1.102 1.688-3.238h1.843l.035.103q-.155 1.757-1.051 3.1-.878 1.344-2.412 2.102-1.515.74-3.6.74z"
            fill="#c084fc"
            style="filter: drop-shadow(1px 1px 2px rgb(0 0 0 / 0.9));"
          />
        </g>
        <g
          transform="matrix(.7033 0 0 .84119 4.282 -13.468)"
        >
          <path
            d="M60.318 59.318H89.681V60.681H60.318z"
            stroke-width="0.637"
          />
          <path
            d="M60.318 69.318H89.681V70.681H60.318z"
            stroke-width="0.637"
          />
          <path
            d="M-79.682 69.318H-50.319V70.681H-79.682z"
            stroke-width="0.637"
            transform="rotate(-90)"
          />
          <path
            d="M-79.682 79.318H-50.319V80.681H-79.682z"
            stroke-width="0.637"
            transform="rotate(-90)"
          />
          <path
            class="animate-logoTicA opacity-0"
            d="M73.147 62.156l1.696 2.112 1.687-2.112h1.446l-2.371 2.901 2.294 2.821h-1.446l-1.61-2.03-1.62 2.03h-1.455l2.294-2.82-2.37-2.902z"
            font-family="Raleway"
            font-size="47.978"
            font-weight="bold"
            id="x5"
            stroke-width="0.097"
          />
          <path
            class="animate-logoTicB opacity-0"
            d="M64.8 77.913q-.75 0-1.368-.242-.607-.242-1.05-.653-.434-.419-.675-.935-.241-.523-.241-1.072 0-.572.25-1.088.26-.523.704-.926.453-.411 1.06-.645.617-.242 1.35-.242.742 0 1.35.25.616.25 1.05.669.434.419.675.935.24.516.24 1.064 0 .564-.25 1.088-.25.515-.704.926-.443.403-1.06.637-.607.234-1.33.234zm-1.985-2.902q0 .371.135.718.135.346.386.62.26.266.626.427.376.154.849.154.482 0 .857-.162.376-.169.627-.443.25-.282.376-.62.135-.347.135-.694 0-.37-.145-.709-.135-.346-.395-.612-.25-.274-.626-.427-.367-.162-.83-.162-.491 0-.867.17-.366.16-.617.435-.25.274-.385.612-.126.339-.126.693z"
            font-family="Raleway"
            font-size="47.978"
            font-weight="bold"
            id="o7"
            stroke-width="0.097"
          />
          <path
            class="animate-logoTicC opacity-0"
            d="M63.119 51.863l1.696 2.111 1.687-2.111h1.446l-2.371 2.901 2.294 2.82h-1.446l-1.61-2.03-1.619 2.03H61.74l2.295-2.82-2.372-2.901z"
            font-family="Raleway"
            font-size="47.978"
            font-weight="bold"
            id="x1"
            stroke-width="0.097"
          />
          <path
            class="animate-logoTicD opacity-0"
            d="M85.183 77.913q-.752 0-1.369-.242-.607-.242-1.05-.653-.434-.419-.675-.935-.24-.523-.24-1.072 0-.572.25-1.088.26-.523.703-.926.453-.411 1.06-.645.618-.242 1.35-.242.742 0 1.35.25.617.25 1.05.669.434.419.675.935.241.516.241 1.064 0 .564-.25 1.088-.251.515-.704.926-.444.403-1.06.637-.608.234-1.33.234zm-1.985-2.902q0 .371.135.718.134.346.385.62.26.266.627.427.376.154.848.154.482 0 .858-.162.376-.169.626-.443.251-.282.376-.62.135-.347.135-.694 0-.37-.144-.709-.135-.346-.396-.612-.25-.274-.626-.427-.366-.162-.83-.162-.49 0-.867.17-.366.16-.617.435-.25.274-.385.612-.125.339-.125.693z"
            font-family="Raleway"
            font-size="47.978"
            font-weight="bold"
            id="o9"
            stroke-width="0.097"
          />
          <path
            class="animate-logoTicE opacity-0"
            d="M73.147 72.11l1.696 2.112 1.687-2.112h1.446l-2.371 2.902 2.294 2.82h-1.446l-1.61-2.03-1.62 2.03h-1.455l2.294-2.82-2.37-2.902z"
            font-family="Raleway"
            font-size="47.978"
            font-weight="bold"
            id="x8"
            stroke-width="0.097"
          />
          <path
            class="animate-logoTicF opacity-0"
            d="M74.829 57.665q-.752 0-1.369-.241-.607-.242-1.05-.653-.434-.42-.675-.935-.241-.524-.241-1.072 0-.572.25-1.088.26-.524.704-.927.453-.41 1.06-.645.617-.241 1.35-.241.742 0 1.35.25.616.25 1.05.668.434.42.675.935.24.516.24 1.064 0 .564-.25 1.088-.25.516-.704.927-.443.403-1.06.637-.607.233-1.33.233zm-1.986-2.901q0 .37.135.717t.386.62q.26.267.626.428.376.153.848.153.482 0 .858-.161.376-.17.627-.443.25-.282.376-.62.135-.347.135-.694 0-.37-.145-.71-.135-.346-.395-.612-.25-.274-.627-.427-.366-.161-.829-.161-.491 0-.867.17-.366.16-.617.434-.25.274-.386.613-.125.338-.125.693z"
            font-family="Raleway"
            font-size="47.978"
            font-weight="bold"
            id="o2"
            stroke-width="0.097"
          />
          <path
            class="animate-logoTicG opacity-0"
            d="M83.501 62.156l1.697 2.112 1.687-2.112h1.445l-2.37 2.901 2.293 2.821h-1.446l-1.61-2.03-1.619 2.03h-1.455l2.294-2.82-2.371-2.902z"
            font-family="Raleway"
            font-size="47.978"
            font-weight="bold"
            id="x6"
            stroke-width="0.097"
          />
          <path
            class="animate-logoTicH opacity-0"
            d="M64.8 67.959q-.75 0-1.368-.242-.607-.242-1.05-.653-.434-.419-.675-.935-.241-.524-.241-1.072 0-.572.25-1.088.26-.523.704-.926.453-.411 1.06-.645.617-.242 1.35-.242.742 0 1.35.25.616.25 1.05.669.434.419.675.935.24.516.24 1.064 0 .564-.25 1.088-.25.515-.704.926-.443.403-1.06.637-.607.234-1.33.234zm-1.985-2.902q0 .371.135.718.135.346.386.62.26.266.626.427.376.154.849.154.482 0 .857-.162.376-.169.627-.443.25-.282.376-.62.135-.347.135-.694 0-.37-.145-.709-.135-.346-.395-.612-.25-.274-.626-.427-.367-.162-.83-.162-.491 0-.867.17-.366.16-.617.435-.25.274-.385.612-.126.339-.126.693z"
            font-family="Raleway"
            font-size="47.978"
            font-weight="bold"
            id="o4"
            stroke-width="0.097"
          />
          <path
            class="animate-logoTicI opacity-0"
            d="M83.501 51.863l1.697 2.111 1.687-2.111h1.445l-2.37 2.901 2.293 2.82h-1.446l-1.61-2.03-1.619 2.03h-1.455l2.294-2.82-2.371-2.901z"
            font-family="Raleway"
            font-size="47.978"
            font-weight="bold"
            id="x3"
            stroke-width="0.097"
          />
        </g>
      </g>
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
